Content.Shared/_Goobstation/MartialArts/SharedMartialArtsSystem.CQC.cs

```diff
@@ -157,16 +162,13 @@ private void OnCQCPressure(Entity<CanPerformComboComponent> ent, ref CqcPressure
             || !TryUseMartialArt(ent, proto.MartialArtsForm, out var target, out _))
             return;
 
+        if (_hands.TryGetActiveItem(target, out var activeItem)) // I know this looks horrible, but the disarm should happen BEFORE the stam dmg, and the popup should always show.
+            if(_hands.TryDrop(target, activeItem.Value))
+                if (_hands.TryGetEmptyHand(target, out var emptyHand))
+                    if(_hands.TryPickupAnyHand(ent, activeItem.Value))
+                        _hands.SetActiveHand(ent, emptyHand);
```

Comment:
```suggestion
        if (_hands.TryGetActiveItem(target, out var activeItem) // I know this looks horrible, but the disarm should happen BEFORE the stam dmg, and the popup should always show.
            && _hands.TryDrop(target, activeItem.Value)
            && _hands.TryGetEmptyHand(target, out var emptyHand)
            && _hands.TryPickupAnyHand(ent, activeItem.Value))
                _hands.SetActiveHand(ent, emptyHand);
```
Double check this, but it's a lit better than if() if() if()

Reply:
Agreed